frontend/src/modules/Core/Components/DropdownSelect.tsx

```diff
@@ -51,6 +50,11 @@ export const DropdownSelect = ({
               },
             },
           },
+          PaperProps: {
+            sx: {
+              borderRadius: '10px',
+            },
+          },
```

Comment:
Was there any reason for adding this here?

Reply:
The previous sx borderRadius for MenuListProps wasn't actually changing what the drop down looked like as it was supposed to